### ISSUE TITLE
TriUtils build failure with Teuchos disabled

### ISSUE THREAD

**Issue by jwillenbring** [trilinos/Trilinos#203]
TriUtils currently has no stated dependence on Teuchos. It does have a stated dependence on Epetra. Epetra has an optional dependence on Teuchos, so Teuchos is typically available. On Sept 30, TriUtils was made dependent on Teuchos, although the dependencies file was not updated.

commit 60dd431da705a59158cecc30055b62b593024824
Author: Nico Schlömer <nico.schloemer@gmail.com>
Date:   Tue Sep 30 11:39:16 2014 +0200

    [TriUtils] char *data_file ==> const char *data_file
    
    Using a `const char*` as function argument allows char literals
    as arguments.
    Besides this change, this commit contains a bunch of indentation corrections
    and replacement of conditional `exit(...)` calls by their TEUCHOS_EXCEPT*
    equivalents.

Basically the options are to make TriUtils depend on Teuchos, or make the Teuchos calls conditional.

@hkthorn - this is affecting the no C++11 nightly build.

**Comment by mhoemmen** [comment]
Looks like the only Teuchos here is as the commit message says: "replacement of conditional `exit(...)` calls by their TEUCHOS_EXCEPT* equivalents," and inclusion of "Teuchos_TestForException.hpp".

**Comment by mhoemmen** [comment]
@nschloe -- it should suffice to depend on TeuchosCore -- don't need all of Teuchos.  The only bit of Teuchos used here lives in TeuchosCore.

**Comment by bartlettroscoe** [comment]
Looks like Nico fixed this.

**Comment by jwillenbring** [comment]
Will change to depend on TeuchosCore like Mark said before closing.

**Comment by mhoemmen** [comment]
Thanks Jim!

**Comment by jwillenbring** [comment]
Pull request issue #207 resolved this issue. I verified in nightly testing.
